pages/new-hackers.md

```diff
@@ -8,3 +8,109 @@ permalink: /new-hackers/
 ### Civic Hacking 101
 
 <iframe width="640" height="360" src="https://www.youtube.com/embed/wH6LnW_qjeI" frameborder="0" gesture="media" allowfullscreen></iframe>
+
+### Getting Started with CodeforNewark
+<ol>
+    <li>
+        <h1> What is Github? </h1>
+        <p> 
+            Github is a platform that is used by millions of developers all over the world to help share their projects and allow others to easily collaborate. On the Codefornewark team, we use Github for this exact purpose.
+        </p>
+        <iframe width="640" height="360" src="https://www.youtube.com/embed/w3jLJU7DT5E" frameborder="0" gesture="media" allowfullscreen></iframe> 
+    </li>
+     <li>
+        <h1>History and Difference between Git and Github</h1>
```

Comment:
I think we could have the Difference between Git and Github paragraph and the Common Git/Github workflow as a link to a separate page. Some of the new Code for Newark members are already familiar with Git and Github.